src/main/java/entities/Postable.java

```diff
@@ -1,4 +1,4 @@
-package entities;
+package java.entities;
```

Comment:
Don't think this is needed. Every entity is in the same directory so we can just use "package entities"

Reply:
This was needed for my intellij to work but I can change it for the repo.